### ISSUE TITLE
Tutorial "Mongo Connection URL" - "Connect tab missing 

### ISSUE THREAD

**Issue by Fishbite** [mdn/content#32075]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment

### What information was incorrect, unhelpful, or incomplete?

The tutorial says "Select the MongoDB service to display information about the database. Open the Connect tab and copy the "Mongo Connection URL" (this is the address of the database). "
![mogodb_interface](https://github.com/mdn/content/assets/43952420/389aa26a-e314-4037-8c13-f916029732e2)

but there is no "Connect" tab when you click the mongdb service:
![mogodb_interface_02](https://github.com/mdn/content/assets/43952420/872b2a45-3003-4ef3-8546-7ce53ae1f1ea)

### What did you expect to see?

I expected to see:
![mogodb_interface](https://github.com/mdn/content/assets/43952420/93a1d415-f02d-47f7-aa06-fe6e6616b8e5)

### Do you have any supporting links, references, or citations?

Here's a link:
[railway](https://railway.app/project/43217fc4-e190-4f68-a011-0618eadcbb94/service/df1d690f-6f97-4d59-a9c8-b92058a24fce)

### Do you have anything more you want to share?

So when I disable the development DB to try to force the app' to use the production DB I get 
```Operation `books.countDocuments()` buffering timed out after 10000ms``` on the app' interface after setting `MONGODB_URI` to equal `MONGO_URL` in the environmental variables 

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/server-side/express_nodejs/deployment`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/express_nodejs/deployment/index.md
* Last commit: https://github.com/mdn/content/commit/206d50cadea3f472d1595c28845b251269f8d6d0
* Document last modified: 2023-12-18T05:17:14.000Z

</details>

**Comment by bsmth** [comment]
Thanks a lot for raising this one @Fishbite - there is a pull request open here to update the images (although it looks stalled): https://github.com/mdn/content/pull/28360

**Comment by hamishwillee** [comment]
THanks. As @bsmth pointed out there is a stalled PR for this. If you'd like to contribute a fix instead of that - that would be great. Otherwise this is on my list to look at at some point. It isn't a huge priority though since it is possible to find the URL with a little searching around mongodb.

**Comment by bsmth** [comment]
Just linking another one here: https://github.com/mdn/content/pull/32381

**Comment by hamishwillee** [comment]
Closing as fixed by #32381

@bsmth - The guide provided by Railway is not excellent and would be hard to follow for our context https://docs.railway.app/guides/mongodb

I am not sure that removing the images is worth it because the churn is quite low - I don't bother updating minor changes - only those where the text and image now are no longer useful. 

But if there does end up being lots of churn in future we might reasonably do this:
- start with the link to the guide and a disclaimer that following content might be out of date.
- then what we have now. Though possibly with fewer images if they can reasonably be removed - such as "New project" dialog

**Comment by bsmth** [comment]
> Closing as fixed by #32381
> 
> @bsmth - The guide provided by Railway is not excellent and would be hard to follow for our context https://docs.railway.app/guides/mongodb


OK, I understand. Glad it's fixed for now - thanks a lot, Hamish 👍🏻
